### DISCUSSION TITLE
hellaswag results are different between opencompass and lm-evaluation-harness

### DISCUSSION THREAD

**Discussion by nlpcat** [open-compass/opencompass#450] [Q&A] [answered]
https://opencompass.org.cn/dataset-detail/HellaSwag

https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard

if we look at the hellaswag in Open LLM leaderboard and opencompass, llama-65b and llama-30b results are different

**Comment by kirliavc** [comment, marked as answer]
The problem is probably due to fewshot inference. The llm leaderboard used 10-shot hellaswag, but the opencompass used 0-shot. Fewshot request is only implemented in mmlu dataset.
